test/WireMock.Net.Tests/RequestWithPathTests.cs

```diff
@@ -14,6 +14,36 @@ public class RequestWithPathTests
     {
         private const string ClientIp = "::1";
 
+        // [Fact] : TODO : this test fails???
```

Comment:
Probably remove comment and TODO?

Reply:
I keep this for now as reminder,

Reply:
But the test doesn't fail anymore right?

Reply:
Somehow this test still fails. Also for netcore.
So for now i've removed the test.

Reply:
Well then the underlying problem (that WireMock.Net does not handle encoded URLs correctly) still persists :(.